doc/amazon_sagemaker_operators_for_kubernetes.rst

```diff
@@ -295,7 +295,285 @@ Your output should look like the following:
 
 
 Verify the operator deployment
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+You should be able to see the Amazon SageMaker Custom Resource
+Definitions (CRDs) for each operator deployed to your cluster by running
+the following command: 
+
+::
+
+    kubectl get crd | grep sagemaker
+
+Your output should look like the following:
+
+::
+
+    batchtransformjobs.sagemaker.aws.amazon.com         2019-11-20T17:12:34Z
+    endpointconfigs.sagemaker.aws.amazon.com            2019-11-20T17:12:34Z
+    hostingdeployments.sagemaker.aws.amazon.com         2019-11-20T17:12:34Z
+    hyperparametertuningjobs.sagemaker.aws.amazon.com   2019-11-20T17:12:34Z
+    models.sagemaker.aws.amazon.com                     2019-11-20T17:12:34Z
+    trainingjobs.sagemaker.aws.amazon.com               2019-11-20T17:12:34Z
+
+Ensure that the operator pod is running successfully. Use the following
+command to list all pods:
+
+::
+
+    kubectl -n sagemaker-k8s-operator-system get pods
+
+You should see a pod
+named \ ``sagemaker-k8s-operator-controller-manager-*****`` in the
+namespace \ ``sagemaker-k8s-operator-system``  as follows:
+
+::
+
+    NAME                                                         READY   STATUS    RESTARTS   AGE
```

Comment:
[Minor] I know this was a copy pate from below, but would it look cleaner if the columns were aligned closer to the values on the second line?
If so, can we make the same change to the section below?

Reply:
Fixed, thanks for the suggestion!